### PR TITLE
Desktop version

### PR DESCRIPTION
**Pull request by jonathanmunamire**
This is the First Pull Request for the Desktop Version

### CODE REVIEW

File: text.html [hunk 1]
```diff
@@ -1,70 +1,84 @@
...
-                    <div ></div>
-                    <div ></div>
-                </nav>
-                
-            </div>
+            <nav class="nav">
+                <div><a href="#">Portfolio</a></div>
+                <div><a href="#About">About</a></div>
+                <div><a href="#Contact">Contact</a></div>
+            </nav>
```
Comment (awais-amjed) [inline text.html:21]: Kindly change the color of these links to match the Figma design 🤗

Your design:

image

Figma design:

image

File: text.html [hunk 2]
```diff
@@ -74,136 +88,198 @@ <h2 class="title">Tonic</h2>
...
-                        <li>
-                            <a href="#"><img src="Linkedinicon.svg"/></a>
-                        </li>
+            </section>
+            <section class="about" id="About">
+                <section class="about1">
+                    <div></div>
+                    <div class="greet">
+                        <h1>About Myself</h1>
+                    </div>
```
Comment (awais-amjed) [inline text.html]: Kindly increase the font size of this heading to match the Figma design, also please note that there should only be one `h1` tag per page so please change this to `h2,h3... etc.` to follow the best practices 🤗

Your design:

image

Figma design:

image

File: text.html [hunk 3]
```diff
@@ -1,70 +1,84 @@
...
+                <div><a href="#About">About</a></div>
+                <div><a href="#Contact">Contact</a></div>
+            </nav>
         </header>
         <section class="headline">
             <div></div>
             <div class="greet">
                 <h1>I'm Jonathan Glad to see you!</h1>
-            </div> 
+            </div>
```
Comment (awais-amjed) [inline text.html:27]: Kindly make sure that your website looks good on mobile version also, so please fix the font-style of this heading to match the Figma design 🤗

Your design:

image

Figma design:

image

File: text.html [hunk 4]
```diff
@@ -74,136 +88,198 @@ <h2 class="title">Tonic</h2>
...
                         </ul>
-                        <p class="description">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+                        <p class="description">Experimental content creation feature that allows users to add to an
+                            existing story over the course of a day without spamming their friends.</p>
                         <ul class="languages">
                             <li class="stack">html</li>
+                            <li class="stack">Ruby on rails</li>
                             <li class="stack">css</li>
                             <li class="stack">javascript</li>
                         </ul>
```
Comment (awais-amjed) [inline text.html:119]: Kindly try to fix this behavior in the mobile version, these tags are coming over the description of the project, kindly fix this for all the projects 🤗

image

**Review by awais-amjed — CHANGES_REQUESTED** [review]
# STATUS: CHANGES REQUIRED ♻️

Hi @jonathanmunamire , 

Good job so far! 👏👏
There are some issues that you still need to work on to go to the next project but you are almost there! 🥳🙌🤗

## Highlights

- [x] Descriptive Pull Request ✔️
- [x] Nice Design ✔️
- [x] Good code organization ✔️ 

gif

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Gopxfs** [comment]
# 🟢 STATUS: APPROVED 🟢

## Hello @jonathanmunamire 👋,

Your project is complete! You followed all the requirements, feel free to merge it! 😄
Congratulations! 🎉 



## Highlights
✔️ Implemented all the changes requested by the previous reviewer.
✔️ Linters are passing.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**